CMakeLists.txt

```diff
@@ -343,37 +354,31 @@ set(POINTMATCHER_SRC
 	pointmatcher/DataPointsFilters/RemoveSensorBias.cpp
 )
 
-include_directories(${CMAKE_SOURCE_DIR})
-
 
 file(GLOB_RECURSE POINTMATCHER_HEADERS "pointmatcher/*.h")
 
-# MacOS and Windows deal with shared/dynamic library differently. For
-# simplicity, we only authorize static library in those cases.
-if(APPLE OR WIN32)
-
-	add_library(pointmatcher ${POINTMATCHER_SRC} ${POINTMATCHER_HEADERS} )
-	install(TARGETS pointmatcher ARCHIVE DESTINATION ${INSTALL_LIB_DIR})
-
-else(APPLE OR WIN32)
-	set(SHARED_LIBS "true" CACHE BOOL "To build shared (true) or static (false) library")
-
-	if (SHARED_LIBS)
-		add_library(pointmatcher SHARED ${POINTMATCHER_SRC} ${POINTMATCHER_HEADERS})
-		install(TARGETS pointmatcher
-						ARCHIVE DESTINATION ${INSTALL_LIB_DIR}
-						LIBRARY DESTINATION ${INSTALL_LIB_DIR}
-						RUNTIME DESTINATION ${INSTALL_BIN_DIR})
-		#install(TARGETS pointmatcher LIBRARY DESTINATION ${INSTALL_LIB_DIR})
-	else(SHARED_LIBS)
-		add_library(pointmatcher ${POINTMATCHER_SRC} ${POINTMATCHER_HEADERS} )
-		install(TARGETS pointmatcher ARCHIVE DESTINATION ${INSTALL_LIB_DIR})
-	endif(SHARED_LIBS)
-endif(APPLE OR WIN32)
-
-target_include_directories(pointmatcher PRIVATE ${CMAKE_SOURCE_DIR}/pointmatcher)
-target_include_directories(pointmatcher PRIVATE ${CMAKE_SOURCE_DIR}/pointmatcher/DataPointsFilters)
-target_include_directories(pointmatcher PRIVATE ${CMAKE_SOURCE_DIR}/pointmatcher/DataPointsFilters/utils)
+
+# In CMake >=3.4 we can easily build shared libraries in Mac and Windows.
+# Don't make any distinction between OSes:
```

Comment:
Cool! 🥳

Reply:
I would rephrase this comment as
'No need to distinguish between operating systems while building targets'

Reply:
Done!